src/modules/bot/commands/wake-up.command.ts

```diff
@@ -9,8 +9,13 @@ export class WakeUpCommand {
 export class WakeUpCommandHandler implements ICommandHandler<WakeUpCommand> {
     async execute(command: WakeUpCommand) {
         if (command.content.length > 0) {
-            const user = command.context.mentions.users.first()
-            await command.context.channel.send(' needs to wake up!', { reply: user })
+            const replies = command.context.mentions.users.map((user) =>
```

Comment:
I wonder if we should just extend the commands to simplify these lines, maybe to allow something like 'command.mentions' which gives you the list of users?

Reply:
while right now they are all the same, maybe we want to give it diff behavior or make it do other things, so idk?